js/graph_config.js

```diff
@@ -736,6 +736,105 @@ GraphConfig.load = function(config) {
                             default:
                                 return getCurveForMinMaxFields(fieldName);
                             }
+                    case 'GPS_RESCUE_THROTTLE_PID':
+                        switch (fieldName) {
+                            case 'debug[0]': // Throttle P
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 500,
+                                    outputRange: 1.0,
+                                };
+                            case 'debug[1]': // Throttle I
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 500,
+                                    outputRange: 1.0,
+                                };
+                            case 'debug[2]': // Throttle D
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 500,
+                                    outputRange: 1.0,
+                                };
+                            case 'debug[3]': // Altitude
+                                return {
+                                    offset: -10000,
+                                    power: 1.0,
+                                    inputRange: 10000,
+                                    outputRange: 1.0,
+                                };
+                            default:
+                                return getCurveForMinMaxFields(fieldName);
+                            }
+                    case 'GPS_RESCUE_VELOCITY':
+                        switch (fieldName) {
+                            case 'debug[0]': // Throttle P
+                                return {
+                                    offset: 0,
+                                    power: 1.0,
+                                    inputRange: 500,
+                                    outputRange: 1.0,
+                                };
+                            case 'debug[1]': // Throttle I
```

Comment:
Best to move this line to 775 (and remove the inner block) as fall through case as the inner block is repeated.

Reply:
OK now I hope